Flint/src/main/java/flint/CLIAdapter.java

```diff
@@ -0,0 +1,67 @@
+package flint;
+
+import java.util.Collection;
+import java.io.FileNotFoundException;
+
+import flint.FlintConfiguration;
+import flint.LintFailure;
+
+public class CLIAdapter {
+    /**
+     * The function that should be triggered by the command line, this will trigger the
+     * linter to check code style based on the user-defined flint config file.
+     *
+     * @param args - should contain the options and parameters from the command line,
+     *               see Overview for details.
+     **/
+    public static void main(String[] args) throws Exception {

```

Comment:
I don't think it makes sense for main() to throw an exception.
The user shouldn't have to look at ugly stacktraces, we should catch everything and output more helpful messages to the user

Reply:
In general, all function docs without mentioning exception will not throw exception, those are just to remind anyone who accidently called it those function are not yet implemented.

Or Should I use another way to express that?

Reply:
No, that's good.
I just didn't realize.

Reply:
Just return something and put a comment about that?